templates/user/settings/keys_ssh.tmpl

```diff
@@ -51,7 +51,7 @@
 			</div>
 			<div class="field {{if .Err_Content}}error{{end}}">
 				<label for="content">{{.i18n.Tr "settings.key_content"}}</label>
-				<textarea id="ssh-key-content" name="content" required>{{.content}}</textarea>
+				<textarea id="ssh-key-content" name="content" placeholder="Begins with 'ssh-rsa', 'ssh-ed25519', 'ecdsa-sha2-nistp256', 'ecdsa-sha2-nistp384', or 'ecdsa-sha2-nistp521'" required>{{.content}}</textarea>
```

Comment:
similarly

Reply:
Would put `ssh-ed25519` first because `ssh-rsa` will be deprecated in the next OpenSSL version.

Reply:
ok, addressed